pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -34,6 +34,10 @@ import (
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
 )
 
+const (
+	kubepods = "kubepods"
```

Comment:
use constant from node_container_manager.go

Reply:
remove this

Reply:
fixed